asn1/LedgerLedgerLotocol.asn

```diff
@@ -124,8 +124,13 @@ CallSet CALL ::= {
 LedgerLedgerLotocolPacket ::= SEQUENCE {
     -- One byte type ID
     type CALL.&typeId ({CallSet}),
-    -- Allow multiple accounts to be multiplexed on one connection
-    accountId UInt32,
+    -- If the to and/or from addresses can be inferred from the
+    -- context (for example, if the connection is a bilateral
+    -- trustline), these values MAY be set to 0-byte strings
+    -- Peers MAY also use shorter aliases to avoid repeatedly
+    -- sending the full ILP Addresses over the wire
+    to Address,
+    from Address,
```

Comment:
I agree that we can expand `accountId` to be zero or more of `toId` and `fromId`,  and I'm also in favor of switching from UInt32 to strings. But why switch to ILP addresses? `accountId` was not an ILP address either, so what changed?
And what should happen when it's a full ILP address, but it's outside the context of this connection? Is the receiver expected to forward it? If not, and like `accountId`, `to[Id]` and `from[Id]` are still strictly tied to the context of the current connection, then I think using ILP addresses here is only confusing.